invenio_mail/ext.py

```diff
@@ -31,7 +31,15 @@
 
 from flask_mail import Mail, email_dispatched
 
-PY3 = sys.version_info[0] == 3
+
+def print_email(message, app):
+    """Print mail to stream."""
+    # Signal handler for email_dispatched signal.
```

Comment:
@JavierDelgadoFernandez this was an example. It should be part of the docstring after extending and reformatting.